controller/api/public/stat_summary_test.go

```diff
@@ -130,7 +131,15 @@ func testStatSummary(t *testing.T, expectations []statSumExpected) {
 			unsortedStatTables := rsp.GetOk().StatTables
 			sort.Sort(byStatResult(unsortedStatTables))
 
-			if !reflect.DeepEqual(exp.expectedResponse.GetOk().StatTables, unsortedStatTables) {
+			okRsp := &pb.StatSummaryResponse{
+				Response: &pb.StatSummaryResponse_Ok_{
+					Ok: &pb.StatSummaryResponse_Ok{
+						StatTables: unsortedStatTables,
+					},
+				},
+			}
+
+			if !proto.Equal(&exp.expectedResponse, okRsp) {
```

Comment:
TIOLI I think you can just compare the `StatSummaryResponse_Ok` messages here, rather than needing to compare the full `StatSummaryResponse` message.

```
			okRsp := &pb.StatSummaryResponse_Ok{
				StatTables: unsortedStatTables,
			}

			if !proto.Equal(exp.expectedResponse.GetOk(), okRsp) {
				t.Fatalf("Expected: %+v\n Got: %+v", &exp.expectedResponse, rsp)
			}
```

Reply:
ah thanks!